### PR TITLE
Iow 523

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ update_configs:
     ignored_updates:
       - match:
           dependency_name: "postgres"
-          version_requirement: "10.7-alpine"
+          version_requirement: "11.8-alpine"
```
Comment (dsteinich) [inline .dependabot/config.yml]: This entire file can be remove now. It was only here to keep dependabot from trying to update postgres.

File: liquibase/changeLogs/capture/indexes/fieldVisitHeaderInfo.yml [hunk 2]
```diff
@@ -7,168 +7,175 @@ databaseChangeLog:
...
-        - sql: create unique index if not exists field_visit_header_info_december_pk on ${AQTS_SCHEMA_NAME}.field_visit_header_info_december (field_visit_header_info_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_december_pk;
-
-  - changeSet:
-      author: ssoper
-      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_december_json_data_id"
-      changes:
-        - sql: create index if not exists field_visit_header_info_december_json_data_id on ${AQTS_SCHEMA_NAME}.field_visit_header_info_december (json_data_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_december_json_data_id;
+      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_json_data_id"
```
Comment (dsteinich) [inline liquibase/changeLogs/capture/indexes/fieldVisitHeaderInfo.yml:10]: We might want to add the partition_number to these indexes as the first column. The name of the indexes can stay the same, but adding the partition_number may improve the query performance by assisting partition pruning(?)

**Review by dsteinich — CHANGES_REQUESTED** [review]
The database/ci/Dockerfile also needs to be updated to PostgreSQL v11

File: liquibase/changeLogs/capture/indexes/fieldVisitHeaderInfo.yml (hunk 2 above)
Comment (ssoper-usgs) [reply liquibase/changeLogs/capture/indexes/fieldVisitHeaderInfo.yml:10]: Good call - I will see about adding the PN to the indexes.

**Review by dsteinich — COMMENTED** [review]
removed

**Review by dsteinich — APPROVED** [review]

**Pull request merged**